libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/ElectronicHorizonParams.kt

```diff
@@ -0,0 +1,13 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation
+
+data class ElectronicHorizonParams @JvmOverloads constructor(
+    val delay: Long = ELECTRONIC_HORIZON_DELAY,
+    val interval: Long = ELECTRONIC_HORIZON_INTERVAL,
+    val locationsCacheSize: Int = LOCATIONS_CACHE_MAX_SIZE
+) {
+    companion object {
+        private const val ELECTRONIC_HORIZON_DELAY = 20_000L
```

Comment:
may be make these `const`s  public. wdyt?

Reply:
should they be public in case of default builder params?

Reply:
It doesn't look like info that we'd like to hide from user. IMO we could keep them public

Reply:
What about leaving them as `private` but changing the name to `_DEFAULT`? Ultimately, these default values are internal and clients can always change them passing other values via the constructor. Don't think should be `public` either.

Reply:
added DEFAULT prefix, saved as private

Reply:
Minor thing @korshaknn let's change to suffix instead so they're better searchable / discoverable.